code_schemes/s01mag10.json

```diff
@@ -3,6 +3,143 @@
   "Name": "S01MAG10",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "DisplayText": "societal benefits",
+      "VisibleInCoda": true,
+      "NumericValue": 0,
+      "CodeID": "code-f5e51184",
+      "CodeType": "Normal",
+      "StringValue": "societal_benefits"
+    },
+     {
+      "DisplayText": "awareness creation",
+      "VisibleInCoda": true,
+      "NumericValue": 1,
+      "CodeID": "code-00b41679",
+      "CodeType": "Normal",
+      "StringValue": "awareness_creation"
+    },
+     {
+      "DisplayText": "equality",
+      "VisibleInCoda": true,
+      "NumericValue": 2,
+      "CodeID": "code-36c12bb6",
+      "CodeType": "Normal",
+      "StringValue": "equality"
+    },
+     {
+      "DisplayText": "encourage girls",
+      "VisibleInCoda": true,
+      "NumericValue": 3,
+      "CodeID": "code-f1c39ec4",
+      "CodeType": "Normal",
+      "StringValue": "encourage_girls"
+    },
+     {
+      "DisplayText": "stop early/forced marriages",
+      "VisibleInCoda": true,
+      "NumericValue": 4,
+      "CodeID": "code-c2a120e8",
+      "CodeType": "Normal",
+      "StringValue": "stop_early_or_forced_marriages"
+    },
+     {
+      "DisplayText": "end belittling girls",
+      "VisibleInCoda": true,
+      "NumericValue": 5,
+      "CodeID": "code-d227fafd",
+      "CodeType": "Normal",
+      "StringValue": "end_belittling_girls"
+    },
+     {
+      "DisplayText": "support girls",
+      "VisibleInCoda": true,
+      "NumericValue": 6,
+      "CodeID": "code-dc9a22db",
+      "CodeType": "Normal",
+      "StringValue": "support_girls"
+    },
+     {
+      "DisplayText": "put an effort",
```

Comment:
put in effort?
put an effort in?

(it just reads slightly weird, without knowing the context of the question at least)

Reply:
Thank you. Updated to put in effort after an offline discussion with rda.